spec/spec_helper.rb

```diff
@@ -36,19 +36,55 @@
     begin
       BW_USERNAME = ENV.fetch("BW_USERNAME")
       BW_PASSWORD = ENV.fetch("BW_PASSWORD")
+      FORBIDDEN_USERNAME = ENV.fetch("BW_USERNAME_FORBIDDEN")
+      FORBIDDEN_PASSWORD = ENV.fetch("BW_PASSWORD_FORBIDDEN")
       BW_ACCOUNT_ID = ENV.fetch("BW_ACCOUNT_ID")
       BW_VOICE_APPLICATION_ID = ENV.fetch("BW_VOICE_APPLICATION_ID")
       BW_MESSAGING_APPLICATION_ID = ENV.fetch("BW_MESSAGING_APPLICATION_ID")
       BASE_CALLBACK_URL = ENV.fetch("BASE_CALLBACK_URL")
       BW_NUMBER = ENV.fetch("BW_NUMBER")
       USER_NUMBER = ENV.fetch("USER_NUMBER")
+      MANTECA_ACTIVE_NUMBER = ENV.fetch("MANTECA_ACTIVE_NUMBER")
+      MANTECA_IDLE_NUMBER = ENV.fetch("MANTECA_IDLE_NUMBER")
+      MANTECA_BASE_URL = ENV.fetch("MANTECA_BASE_URL")
+      MANTECA_APPLICATION_ID = ENV.fetch("MANTECA_APPLICATION_ID")
+      OPERATING_SYSTEM = ENV.fetch("OPERATING_SYSTEM")
+      RUBY_VERSION = ENV.fetch("RUBY_VERSION")
     rescue
       puts "Environmental variables not found"
       exit(-1)
     end
 
     DATA = 0    # index for response array related to the response data
     CODE = 1    # index for response array related to the status code
+    SLEEP_TIME_S = 3  # default sleep time in seconds
+    MAX_RETRIES = 40
+    $active_calls = []
+
+    $complete_call_body = Bandwidth::UpdateCall.new(
+      state: Bandwidth::CallStateEnum::COMPLETED
+    )
+  }
+
+  config.after(:suite) {
+    Bandwidth.configure do |config|
+      config.username = BW_USERNAME
+      config.password = BW_PASSWORD
+    end
+    calls_api = Bandwidth::CallsApi.new
+    
+    begin
+      $active_calls.each do |call_id|
+        response = calls_api.get_call_state(BW_ACCOUNT_ID, call_id)
+        if !(response.state == 'disconnected')
+          calls_api.update_call(BW_ACCOUNT_ID, call_id, $complete_call_body)
+        end
+      end
+    rescue Bandwidth::ApiError => e
+      puts e.inspect
+      error_message = "Failed to terminate all calls" + $active_calls.to_s
```

Comment:
Do we want to do a polling retry behavior?  This was actually a somewhat common failure for "call with id <id>" was not found in the python sdk.  What are the consequences of a call not being terminated?